cmd/largeUpload.go

```diff
@@ -0,0 +1,35 @@
+//go:build wip
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/storacha/guppy/cmd/internal/largeupload"
+)
+
+var largeUploadCmd = &cobra.Command{
+	Use:   "large-upload",
```

Comment:
looks like this expects a single argument(?)
`Args:  cobra.ExactArgs(1),` is nice for this.

Edit: I see it's already be used elsewhere, nice!